cosmic_ray/worker.py

```diff
@@ -14,6 +14,13 @@
 import traceback
 
 import astunparse
+try:
+    import typing      # the typing module does some fancy stuff at import time
+                       # which we shall not do twice... by loading it here,
+                       # preserve_modules does not delete it and therefore
+                       # fancy stuff happens only once
+except:
```

Comment:
What is the exception that you expect here? The only exception I would expect here is an `ImportError` for Python 3.4 (the earliest version we support).